pkgs/tools/misc/zf/default.nix

```
@@ -0,0 +1,51 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  zig,
+  testers,
+  zf,
+  ...
+}:
+stdenv.mkDerivation rec {
+  pname = "zf";
+  version = "0.8.0";
+
+  src = fetchFromGitHub {
+    owner = "natecraddock";
+    repo = pname;
+    rev = version;
+    fetchSubmodules = true;
+    hash = "sha256-MzlSU5x2lb6PJZ/iNAi2aebfuClBprlfHMIG/4OPmuc=";
+  };
+
+  nativeBuildInputs = [zig];
+
+  buildPhase = ''
+    runHook preBuild
+
+    export HOME="$TMPDIR"
+    zig build -Drelease-safe
```

```suggestion
    zig build -Drelease-safe -Dcpu=baseline
```

Needs baseline to maintain compatibility.

---

done